src/track/playcounter.h

```diff
@@ -1,22 +1,19 @@
 #ifndef MIXXX_PLAYCOUNTER_H
 #define MIXXX_PLAYCOUNTER_H
 
+#include "control/controlproxy.h"
 #include "util/assert.h"
 
-
 // Counts the total number of times a track has been played
 // and if the track has been played during the current session.
 class PlayCounter {
 public:
-    explicit PlayCounter(int timesPlayed = 0):
-        m_iTimesPlayed(timesPlayed),
-        m_bPlayed(false) {
-    }
+  explicit PlayCounter(int timesPlayed = 0);
 
-    // Sets total number of times a track has been played
-    void setTimesPlayed(int iTimesPlayed) {
-        DEBUG_ASSERT(0 <= iTimesPlayed);
-        m_iTimesPlayed = iTimesPlayed;
+  // Sets total number of times a track has been played
```

Comment:
Please revert this indentation change.
Set you editor to use four space as Tabs.

I also recommend to use git-clang-format to do the code style right. See:
https://www.mixxx.org/wiki/doku.php/coding_guidelines
It just needs some help breaking long lines at colum 80 ... 100.

Reply:
This reformate was introduced by git-clang-format. I was also suprised by it. Will remove this, indention change.